packages/storybook/stories/react-inbox/with-toast.stories.tsx

```diff
@@ -10,8 +10,8 @@ export default {
   args: {},
 };
 
-const clientKey = "ZDRiMTViNmMtODdmZi00OWMzLWI5YTgtYmU5NTc3YTRlM2Y1";
-const userId = "3b17362c-0c91-4fcd-a829-885a2343eb67";
+const clientKey = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
```

Comment:
we should find a way to use env variables

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/9949a47a-00d0-457e-8ebe-b3ad10f2c7de-ca6dcc55-6972-48eb-ab46-ee645034caa6-v1.png?transparent=1&palette=1&width=246)